### ISSUE TITLE
HELP NEEDED!!!!

### ISSUE THREAD

**Issue by Impulse706** [XorTroll/Goldleaf#714]
I'm using the latest version of everything and can't get my installed titles to launch. I'm stuck with a loading icon that if click on, says "Could not start the software". Can anyone help?? Anything would be very much appreciated.

**Comment by lopww** [comment]
Not sure but many people use the likes of [DBI](https://github.com/rashevskyv/dbi) instead, try that.

**Comment by V-Karch** [comment]
@Impulse706 Have you updated sigpatches and firmware?

**Comment by Impulse706** [comment]
Yes, I am using all of the latest versions.

**Comment by V-Karch** [comment]
@lilfmdude thank you for that warning tbh, I had no idea about this. I've been recommending DBI to people because it's never had issues for me, so I'll have to talk to all of them about this now

**Comment by Soltinator** [comment]
Its sigpatches. 
as far as i recal, sigpatches had some issues with the atmosphere and tesla overlay about automatic launching or something. Not a goldleaf issue. You can close this.

**Comment by XorTroll** [comment]
Yeah I'm pretty sure this is sigpatches, such an issue is surely not from Goldleaf itself
